apps/wallet/src/ui/app/hooks/index.ts

```diff
@@ -21,5 +21,9 @@ export { useTransactionDryRun } from './useTransactionDryRun';
 export { useTransactionSummary } from './useTransactionSummary';
 export { useRpc } from './useRpc';
 export { useGetObject } from './useGetObject';
+export {
+    useGetTransactionById,
+    useGetTransactionIdByAddress,
+} from './useGetTransaction';
```

Comment:
Don't love filenames that imply exports but don't actually export the implied export. Would be fine if this was named `transactions`.